packages/evolution-generator/src/tests/test_generate_section_configs.py

```
@@ -0,0 +1,234 @@
+    add_generator_comment,
+    create_mocked_excel_data,
+    delete_file_if_exists,
+)
+from scripts.generate_section_configs import (
+    _section_conditional_ts_expression,
+    generate_section_configs,
+)
+
+MOCKED_EXCEL_FILE = "src/tests/references/test.xlsx"
+
+I = INDENT
```

Ruff (0.15.12)

[error] Ambiguous variable name: `I`

(E741)